lib/active_campaign/api/contact_tags.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ActiveCampaign
+  module API
+    #
+    # Interface to contact tag endpoints
+    #
+    # @author Tim Chaston <timchaston@gmail.com>
+    #
+    module ContactTags
+      #
+      # Add a tag to a contact
+      #
+      # @param [Hash] params add a tag to a contact with this data
+      # @param params [String] :contact ID of the contact you're adding the tag to
+      # @param params [String] :tag ID of the tag to be added for the contact
+      #
+      # @return [Hash] a hash with the information of the newly created contact tag
+      #
+      def create_contact_tag(params)
+        post('contactTags', contactTag: params)
```

Comment:
please use `contact_tag` as the case manipulation is handled by a middleware and I don't like Java notation in ruby.